common/utils/src/main/resources/error/error-classes.json

```diff
@@ -75,6 +75,16 @@
       }
     }
   },
+  "AVRO_INCOMPATIBLE_READ_TYPE" : {
+    "message" : [
+      "Cannot convert Avro <avroPath> to SQL <sqlPath> because the original encoded data type is <avroType>, however you're trying to read the field as <sqlType>, which would lead to an incorrect answer. To allow reading this field, enable the SQL configuration: \"spark.sql.legacy.avro.allowIncompatibleSchema\"."
+    ]
+  },
+  "AVRO_LOWER_PRECISION" : {
```

Comment:
isn't it also `AVRO_INCOMPATIBLE_READ_TYPE`?

Reply:
I still feel like these two error type should be separate. For `AVRO_LOWER_PRECISION`, we should just provide a wider decimal type. For `AVRO_INCOMPATIBLE_READ_TYPE`, we should provide a correct read type. It's different behavior.

Reply:
sounds good. Let me unify these two error type.